### PR TITLE
Mars

### PR DESCRIPTION
**Pull request by devaigergely81**
This branch works on Eclipse Mars with Papyrus 1.1.2.
@dobreffandras, @gerazo, please review!

### CODE REVIEW

**Comment by gerazo** [comment]
Pretty perfect for me.

File: releng/hu.elte.txtuml.target/mirrors.target [hunk 1]
```diff
@@ -1,37 +1,29 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde version="3.8"?>
 <target name="mirrors" sequenceNumber="8">
-	<!-- TODO: change these to mirror sites for speeding up development -->
+	<!-- TODO: upload the same versions to the mirror site and change the locations in this file -->
```
Comment (gerazo) [inline releng/hu.elte.txtuml.target/mirrors.target]: Could you do this @kmate ?

Comment (kmate) [inline releng/hu.elte.txtuml.target/mirrors.target]: Yes, I am working on it... Basically I re-mirror 3 update sites to have the latest versions of these components. If I am right, xtext components could also be found in the mars repository, so we will not need a separate tmf update site.

**Comment by dobreffandras** [comment]
Code review - Modifications on the export.papyrus: OK
Manual Test - Generating diagrams from examples: OK
Unit Tests - export.papyrus.test: All running green: OK
(Note: Mars is slower and presents the same "stability" as Luna - based on experience)

**Comment by kmate** [comment]
Mirrors are ready and working.

**Comment by gerazo** [comment]
Super. Thank you all!

**Pull request merged**